src/app/courses/progress-courses/courses-progress-chart.scss

```diff
@@ -2,7 +2,7 @@
 
 :host {
 
-  $cell-size: 45px;
+  $cell-size: 71px;
```

Comment:
This leaves too much space around the numbers in the chart.  We should either abbreviate question or maybe rotate the text to fit.